src/pretix/control/forms/event.py

```diff
@@ -985,10 +985,28 @@ class MailSettingsForm(SettingsForm):
         widget=I18nTextarea,
     )
     smtp_use_custom = forms.BooleanField(
-        label=_("Use custom SMTP server"),
-        help_text=_("All mail related to your event will be sent over the smtp server specified by you."),
+        label=_("Use Custom Email"),
```

Comment:
I think better to say: "Use custom email provider".
Please don't capitalize unnecessarily.